nestor_api/lib/k8s/builders.py

```diff
@@ -41,6 +64,143 @@ def load_templates(templates_path: str, template_names: list) -> dict:
     }
 
 
+def get_sections_for_process(
+    process: dict, deployment_config: dict, tag_to_deploy: str, templates: dict
+) -> list:
+    """Build the deployment sections of a process."""
+    process_name = process["name"]
+    app_name, sanitized_process_name, metadata_name = get_sanitized_names(
+        deployment_config, process_name
+    )
+    image_name = get_image_name(deployment_config, {"tag": tag_to_deploy})
+
+    deployment_sections = []
+
+    if process_name == "web":
+        web_service = yaml_lib.parse_yaml(
+            templates["service"](
+                {
+                    "app": app_name,
+                    "name": app_name,
+                    "image": image_name,
+                    "target_port": K8sConfiguration.get_service_port(),
+                    **deployment_config.get("templateVars", {}),
+                }
+            )
+        )
+        deployment_sections.append(web_service)
+
+    deployment = yaml_lib.parse_yaml(
+        templates["deployment"](
+            {
+                "app": app_name,
+                "name": metadata_name,
+                "image": image_name,
+                "process": sanitized_process_name,
+                "project": deployment_config["project"],
+                **deployment_config.get("templateVars", {}),
+            }
+        )
+    )
+
+    timestamp = round(time.time() * 1000)  # timestamp in milliseconds
+    deployment["spec"]["template"]["metadata"]["annotations"] = {"date": str(timestamp)}
+
+    set_secret(deployment_config, deployment)
+
+    hpa = set_replicas(deployment_config, process_name, deployment, templates)
+    if hpa:
+        deployment_sections.append(hpa)
+
+    set_anti_affinity(deployment_config, process_name, deployment, templates)
+
+    set_node_selector(deployment_config, process_name, deployment)
+
+    set_resources(deployment_config, process_name, deployment)
+
+    set_command(process, deployment)
+
+    set_probes(deployment_config, process_name, deployment)
+
+    deployment["spec"]["template"]["spec"]["containers"][0]["ports"] = [
+        {"containerPort": K8sConfiguration.get_service_port()}
```

Comment:
Maybe have this called once at the beginning of the function
```suggestion
       service_port = K8sConfiguration.get_service_port()
```